frontend/package-lock.json

```diff
@@ -7236,8 +7236,9 @@
       }
     },
     "node_modules/axios": {
-      "version": "1.3.5",
-      "license": "MIT",
+      "version": "1.6.2",
```

Comment:
Shouldn't this have been included in the release?

Reply:
Yes, but then I need to open two PR's like last time, I was not sure how to do this properly.

Reply:
Yes, two PRs are usually needed (pre and post release).